### ISSUE TITLE
Regarding the relation between the hydration system and Comfort effect in Farmer's delight.

### ISSUE THREAD

**Issue by unrealpenguin139** [ghen-git/Thirst-Mod#104]
Regarding comfort effect in Farmer's delight, is it intended to last only when the hydration is full?
Also, is it possible to set it so that the effect lasts even when the hydration is not full (as I think comfort is originally intended to be such an effect)?

**Comment by mlus-asuka** [comment]
Nope. Thirst will just simply stop decreasing when have comfort effect.
